__tests__/app.test.js

```diff
@@ -79,6 +79,46 @@ describe("/api/articles/:article_id", () => {
       });
   });
 });
+
+describe("/api/articles", () => {
+  test("GET 200: Responds with array of all article objects.", () => {
+    const exampleArticle = {
+      title: expect.any(String),
+      topic: expect.any(String),
+      author: expect.any(String),
+      votes: expect.any(Number),
+      created_at: expect.any(String),
+      article_img_url: expect.any(String),
+      article_id: expect.any(Number),
+      comment_count: expect.any(Number),
+    };
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(articles.length).toBe(13);
+        articles.forEach((article) => {
+          expect(article).toMatchObject(exampleArticle);
+        });
+      });
+  });
+
+  test("GET 200: Responds with all article objects sorted by date in descending order", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(articles).toBeSortedBy("created_at", { descending: true });
+        articles.forEach((article) => {
+          expect(article).not.toHaveProperty("body");
+        });
+        
```

Comment:
Great testing, well done for following the requirements of the task with the ordering and excluding the body.